systemd/bootupd.service

```diff
@@ -11,7 +11,17 @@ StartLimitBurst=10
 Type=notify
 Environment=BOOTUPD_VERBOSITY="-v"
 ExecStart=/usr/libexec/bootupd daemon $BOOTUPD_VERBOSITY
-# On general principle
+# This way our working directory isn't writable by default.
+WorkingDirectory=/usr/share/empty
```

Comment:
Apparently this is a RPM-specific directory introduced in 2000 (but I couldn't find more info on that).

Neither LHS nor a Debian machine have it. I don't know how broad is the "distribution-independent" bit in the README, but this likely won't work on non-RPM systems.

Reply:
I'd say non-Fedora systems (it's not a function of the package format so much as the `filesystem` package).  But yes.  I guess we could make our own.   It'd be really unfortunate to add a `#[cfg]` for this.

Eh, I just changed it to use `/usr` since we care more about read-only then we can about empty.